deepchem/molnet/load_function/kt_dataset.py

```diff
@@ -0,0 +1,210 @@
+    transformers : list
+      ``deepchem.trans.transformers.Transformer`` instances applied
+      to dataset.
+
+  References
+  ----------
+  MLA style references for the KT parameter dataset. 
+    [1] Marcus, Yizhak. "The properties of organic liquids that are relevant to their use as solvating solvents." Chemical Society Reviews 22.6 (1993): 409-416.
+  Examples
+  --------
+  >> import deepchem as dc
+  >> tasks, datasets, transformers = dc.molnet.load_kt_dataset(reload=False)
+  >> train_dataset, val_dataset, test_dataset = datasets
+  >> n_tasks = len(tasks)
+  >> n_features = train_dataset.get_data_shape()[0]
+  >> model = dc.models.MultitaskClassifier(n_tasks, n_features)
+"""
+
+
+  
+  
+
+  # Featurize ktdataset
+  logger.info("About to featurize ktdataset.")
+  kt_tasks = ["alpha", "beta"]  # machine learning targets
+
+  # Get DeepChem data directory if needed
+  if data_dir is None:
+    data_dir = DEFAULT_DIR
+  if save_dir is None:
+    save_dir = DEFAULT_DIR
+
+  # Check for str args to featurizer and splitter
+  if isinstance(featurizer, str):
+    featurizer = DEFAULT_FEATURIZERS[featurizer](**featurizer_kwargs)
+  elif issubclass(featurizer, Featurizer):
+    featurizer = featurizer(**featurizer_kwargs)
+
+  if isinstance(splitter, str):
+    splitter = DEFAULT_SPLITTERS[splitter]()
+  elif issubclass(splitter, Splitter):
+    splitter = splitter()
+
+  # Reload from disk
+  if reload:
+    featurizer_name = str(featurizer.__class__.__name__)
+    splitter_name = str(splitter.__class__.__name__)
+    save_folder = os.path.join(save_dir, "ktdataset-featurized",
+                               featurizer_name, splitter_name)
+
+    loaded, all_dataset, transformers = deepchem.utils.save.load_dataset_from_disk(
+        save_folder)
+    if loaded:
+      return kt_tasks, all_dataset, transformers
+
+  # First type of supported featurizers
+  supported_featurizers = ['ECFP', 'GraphConv', 'Weave', 'Raw', 'AdjacencyConv']  # type: List[Featurizer]
+
+  # If featurizer requires a non-CSV file format, load .tar.gz file
+  if featurizer in supported_featurizers:
+    dataset_file = os.path.join(data_dir, 'KTparameterDataset.csv')
+
+    if not os.path.exists(dataset_file):
+      deepchem.utils.download_url(url=KTDATASET_URL, dest_dir=data_dir)
+      deepchem.utils.untargz_file(
+          os.path.join(data_dir, 'ktdataset.tar.gz'), data_dir)
+
+    # Changer loader to match featurizer and data file type
+    loader = deepchem.data.DataLoader(
+        tasks=kt_tasks,
+        id_field="cid",  # column name holding sample identifier
+        featurizer=featurizer)
+
+  else:  # only load CSV file
+    dataset_file = os.path.join(data_dir, "ktdataset.csv")
+    if not os.path.exists(dataset_file):
+      deepchem.utils.download_url(url=KTDATASET_CSV_URL, dest_dir=data_dir)
+
+    loader = deepchem.data.CSVLoader(
+        tasks=kt_tasks, smiles_field="csmiles", featurizer=featurizer)
```

Comment:
Could you use `feature_field` instead of `smiles_field`? 

```
loader = deepchem.data.CSVLoader(
    tasks=kt_tasks, feature_field="csmiles", featurizer=featurizer)
```

Reply:
Thanks. I made the change accordingly.